meetup-checklist.md

```diff
@@ -0,0 +1,82 @@
+Meetup Checklist
+================
+
+**NOTE**: This is our template for organizing a meetup and making sure we get
+all the little details covered! When we're starting a new meetup, please
+[open a new issue](https://github.com/seanode/meetup/issues) with a title
+like `Meetup - DATE - TOPICS`, paste this template in, remove this boilerplate
+section you're reading right now, and tweak specific to the details of the
+event.
+
+## Pre-event
+
+* [ ] Get a venue host.
+    * [ ] Confirm **maximum seating capacity** and enter in meetup RSVP limit.
+
+* [ ] Get a food / beverages host _or_ confirm venue host will provide.
+    * [ ] Confirm _what_ is actually going to be served (e.g., snacks vs. dinner).
+
+* [ ] Get first speaker.
+    * [ ] Get title / abstract / bio / twitter handle.
+    * [ ] Confirm talk length (20-30 mins) and limiting commercial content.
```

Comment:
I can't stress this enough: "limiting commercial content". The last meetup felt like an advertisement for surface and azure!